container/gitpod/Dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM opensuse/leap:15.3
+
+# only dependencies for CircleCI to be able to load/save the package cache
+# and fix permissions for the unprivileged user we use
+RUN zypper -n in tar gzip sudo system-group-wheel && zypper clean
+
+# these are autoinst dependencies
+RUN zypper install -y gcc-c++ cmake ninja pkgconfig\(opencv\) pkg-config perl\(Module::CPANfile\) pkgconfig\(fftw3\) pkgconfig\(libpng\) pkgconfig\(sndfile\) pkgconfig\(theoraenc\) npm
+
+# openQA dependencies
+RUN zypper install -y rubygem\(sass\) python3-base python3-requests python3-future git-core rsync curl postgresql-devel postgresql-server qemu qemu-kvm qemu-tools tar xorg-x11-fonts sudo make
```

Comment:
I don't want to maintain this package list manually here as well. You need to find a better solution. At least for gitpod, can you try openQA-devel instead of all packages manually?